### ISSUE TITLE
 File '.' does not exist (any more).

### ISSUE THREAD

**Issue by nulltoken** [danielpalme/ReportGenerator#80]
Running 2.5.3-beta2 version and passing the program a relative path to the `reports/targetdir` arguments trigger a warning.

```powershell
$coverageFolder = "./coverage"

& $reportgenerator `
 -targetdir:$coverageFolder `
  -reporttypes:Html `
  -reports:$coverageFolder/Cobertura.coverage.xml `
  -verbosity:Verbose
```

generates

```
Loading report 'F:\Dummy\.\coverage\Cobertura.coverage.xml'
 Preprocessing report
 Initiating parser for Cobertura
  Current Assembly: XXX
Initializing report builders for report types: Html
Analyzing 64 classes
 Creating report 1/64 (Assembly: aaaaa)
...
 Creating report 13/64 (Assembly: bbbbb)
 File '.' does not exist (any more).
 Creating report 14/64 (Assembly: ccccc)
...
 Creating report 64/64 (Assembly: ddddd)
 Creating summary
Report generation took 0.7 seconds
```

Maybe normalizing the generated rooted paths would avoid this?

**Comment by danielpalme** [comment]
Just use an unrooted path should fix the problem:

```powershell
$coverageFolder = "coverage"
```

Does this help?

**Comment by nulltoken** [comment]
>Using an unrooted path should fix the problem:

Yes, of course. And even the warning isn't blocking as the console app doesn't return any non zero exit code when that happens.

This issue was just to raise the point that maybe the tool could sanitize the paths before processing them.

However, if you think this is irrelevant, feel free to close this feature request. :wink:

**Comment by danielpalme** [comment]
I took a closer look at the console output you posted.
The only warning I can find is:
```
File '.' does not exist (any more).
```

But that has nothing to do with the rooted path argument you pass to _ReportGenerator_.

Please reopen if I missed something.

**Comment by nulltoken** [comment]
Thanks for having taken a look.

I'm somewhat puzzled then. I assumed this dot was coming from `Loading report 'F:\Dummy\.\coverage\Cobertura.coverage.xml'`. 

>But that has nothing to do with the rooted path argument you pass to ReportGenerator.

Where did I mess up? Any hint?

**Comment by danielpalme** [comment]
The '.' probably comes from the Cobertura file.
The file contains a <sources> element. Most likely it contains an entry '.'

**Comment by nulltoken** [comment]
I've taken a look at it. It doesn't.

Any other lead?

```
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE coverage SYSTEM "http://cobertura.sourceforge.net/xml/coverage-04.dtd">
<coverage line-rate="0.781150159744409" branch-rate="0.678638941398866" lines-covered="978" lines-valid="1252" branches-covered="359" branches-valid="529" complexity="0" version="0" timestamp="1486935257">
  <sources>
    <source>F:\path\to\directory_containing_sln</source>
  </sources>
  <packages>
       [...]
  </packages>
</coverage>
```

FWIW, the converter is invoked this way

```
Run { & $coberturaConverter `
    `"-input:$root/coverage.opencover.xml`" `
    `"-output:$root/coverage.cobertura.xml`" `
    `"-sources:$root`"
}
```

`Run` being a simple function that executes the script block and the throws if `LASTEXITCODE` isn't equal to 0

**Comment by danielpalme** [comment]
Sorry, I don't know why.

**Comment by nulltoken** [comment]
No problem. As you've released the fix for #79, I'll try and find the time to isolate the issue.
